include/boost/dynamic_bitset/dynamic_bitset.hpp

```diff
@@ -1486,29 +1487,32 @@ dynamic_bitset<Block, Allocator>::find_first() const
     return m_do_find_from(0);
 }
 
-
 template <typename Block, typename Allocator>
 typename dynamic_bitset<Block, Allocator>::size_type
-dynamic_bitset<Block, Allocator>::find_next(size_type pos) const
+dynamic_bitset<Block, Allocator>::find_first(size_type offset) const
 {
-
     const size_type sz = size();
-    if (pos >= (sz-1) || sz == 0)
```

Comment:
The logic for `sz == 0` was eliminated.   Was that intentional?

Reply:
`sz == 0` was necessary because of the `sz-1`. the new check is `pos >= sz`.